tests/java-example-auth.yml

```diff
@@ -0,0 +1,19 @@
+---
+runner: command
+dir: java-example/java
+args: 
+  - mvn
+  - compile
+  - exec:java
+  - -Dexec.mainClass=com.start.Connection
+  - -Dexec.args="mongodb://user:password@localhost:27017/?authMechanism=PLAIN"
+
+results:
+  ferretdb:
+    stats:
+      expected_pass: 1
+
+  mongodb:
+    stats:
+      # PLAIN is used in MongoDB to perform LDAP authentication.
+      unexpected_fail: 1
```

Comment:
`unexpected_fail` should not be expected :)

Reply:
:+1: